packages/opencensus-core/src/stats/stats.ts

```diff
@@ -188,8 +188,8 @@ export class BaseStats implements Stats {
     }
 
     if (!tags) {
-      // TODO(mayurkale): read tags current execution context
-      tags = new TagMap();
+      // Record against implicit (current) context
+      tags = getCurrentTagMap();
```

Comment:
Will users be able to have fine-grained control over which tags for a given metric come from the context and which should be explicitly specified? Or over which tags in the context are associated with which metric?

Reply:
No we intentionally don't provide that level of control. Tags in the context are treated as "opaque" to prevent users accidentally leak confidential information. "Opaque" here means users can insert or upsert tags into context, but they cannot see what tags are in current context or iterate over them.

Reply:
OK. Is it possible to have a case where a metric does not have labels associated with all the tags in the context?

Reply:
Yes, it is possible. This is the logic to handle that ->  https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/stats/recorder.ts#L47

Reply:
Makes sense, thanks